docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -54,7 +54,10 @@ \subsection{Stakeholders}
 
 \subsection{Environment}
 
-\wss{Hardware and software}
+\subsubsection{Hardware} 
+Any device with cameras and screen
+\subsubsection{Software}
+Database server for storing and aggregating data
```

Comment:
Can we format these as full sentences?

You also forgot to put your date, name and changes in the changelog at the top.

i.e. "The software should be used on any device with a camera and screen"

Reply:
Resolved, looking for further review.